style.css

```diff
@@ -0,0 +1,989 @@
+}
+
+.after-image-image-4 {
+  background-image: url("./images/product_29456_product_shots1.jpg");
+}
+
+.group-picture-5 {
+  background-image: url("./images/Social-media-hed-image-796x419.jpg");
+}
+
+.after-image-image-5 {
+  background-image: url("./images/product_34441_product_shots1.jpg");
+}
+
+.group-picture-6 {
+  background-image: url("./images/Facebook-Discover-796x419.jpg");
+}
+
+.after-image-image-6 {
+  background-image: url("./images/product_35414_product_shot_wide.jpg");
+}
+
+.group-picture-7 {
+  background-image: url("./images/BMW-ix3-unveiling-car-future-bavarian-motor-ev-electric-796x448.png");
+}
+
+.after-image-image-7 {
+  background-image: url("./images/product_38387_product_shots1.jpg");
+}
+
+.group-picture-8 {
+  background-image: url("./images/1.jpg");
+}
+
+.after-image-image-8 {
+  background-image: url("./images/product_41751_product_shots1.jpg");
+}
+
+.group-picture-9 {
+  background-image: url("./images/Legend-of-Korra-796x417.jpg");
+}
+
+.after-image-image-9 {
+  background-image: url("./images/product_41895_product_shots1.jpg");
+}
+
+.top-grid {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-column-gap: 10px;
+  padding: 10px;
+}
+
+.top-grid-1 {
+  background-image: url("./images/9.jpeg");
+  width: 100%;
+  background-position: 50%;
+  background-size: cover;
+}
+
+.top-grid-2 {
+  background-image: url("./images/1.jpg");
+  width: 100%;
+  background-position: 50%;
+  background-size: cover;
+  margin-bottom: 10px;
+}
+
+.top-grid-3 {
+  background-image: url("./images/8.jpeg");
+  width: 100%;
+  background-position: 50%;
+  background-size: cover;
+}
+
+.child-sec {
+  background: linear-gradient(rgba(255, 0, 0, 0), rgba(0, 0, 0, 1));
+  position: relative;
+  padding-top: 40%;
+  height: 3%;
+  background-size: cover;
+  width: 100%;
+}
+
+.head-text {
+  position: absolute;
+  bottom: 0;
+  margin: 10px;
+}
+
+.head-text-title {
+  color: white;
+  font-size: 45px;
+  margin-bottom: 10px;
+}
+
+.head-text-bottom {
+  color: #555;
+  font-size: 12px;
+}
+
+.headlines {
+  color: white;
+  font-size: 26px;
+}
+
+.headings {
+  color: white;
+  font-size: 16px;
+}
+
+.handi {
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  margin: 20px;
+  border-radius: 50%;
+  background-color: blue;
+  color: white;
+  font-size: 28px;
+  padding: 10px;
+  z-index: 10;
+}
+
+.deal-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.deals-right {
+  color: #f42;
+}
+
+.nav-img {
+  background-color: red;
+  padding: 11px 20px;
+  height: 24px;
+  width: 66px;
+  margin: 0;
+  position: absolute;
+}
+
+#overwrite {
+  margin: 0;                                                                                                              
+}
+
+.nav-2-left-holder {
+  display: flex;
+}
+
+.hidden-1,
+.hidden-2 {
+  display: none;
+}
+
+#hidden-2 {
+  display: none;
+}
+
+@media only screen and (max-width: 1024px) {
+  .latest-news-sec {
+    grid-template-columns: repeat(3, 1fr);
+  }
```

Comment:
- [ ] Your breakpoint for large screen starts on 1025px, should be 1024px. Please change to `@media only screen and (max-width: 1023px) {}` to achieve this

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| ![image](https://user-images.githubusercontent.com/24902525/89531830-77136180-d7b6-11ea-9609-e6bb0582f516.png) | ![image](https://user-images.githubusercontent.com/24902525/89531811-6b279f80-d7b6-11ea-8894-e657ce2a73c3.png) |